webapp/javascript/components/TimelineChart/TimelineChartWrapper.tsx

```diff
@@ -193,29 +202,28 @@ class TimelineChartWrapper extends React.Component<
       // to use custom apperance and color for it
       const boundary = {
         lineWidth: 1,
-        color:
-          this.props.selectionType === 'double' ? m.color.rgb() : 'transparent',
+        color: m.color.rgb(),
```

Comment:
this line breaks selection appearance in a bit
![image](https://user-images.githubusercontent.com/7372044/188097190-6c010b7a-5262-4245-bc2b-6936fa4ef2dd.png)
previous one works as expected:
![image](https://user-images.githubusercontent.com/7372044/188097087-1a95bd29-bab9-4bb9-b5dc-70e58121658a.png)

Reply:
ahh you are absolutely right